modules/ROOT/pages/apply-configure-json-threat-task.adoc

```diff
@@ -9,17 +9,23 @@ endif::[]
 >s|Policy Name|JSON Threat Protection
 >s|Summary|Protects against malicious JSON in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
+JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON Threat Protection policy helps protect your applications from such intrusions.
```

Comment:
"JSON requests are susceptible to attacks" The problem is not on the requests themselves but on the applications processing those requests. Something more accurate would be: Application consuming JSON based request are susceptible to attacks… Or something in those lines.

Reply:
Done! Thanks!